lobby/build.gradle

```diff
@@ -134,23 +126,3 @@ checkstyleTest {
     maxWarnings = 0
     source sourceSets.test.output.resourcesDir
 }
-
-afterEvaluate {
-    def junitPlatformTest = tasks.junitPlatformTest
-
-    jacoco {
```

Comment:
Are we still generating test coverage reports after this?

Reply:
I just checked the reports are still getting generated on my machine, but travis doesn't seem to pick it up for some reason

Reply:
Ok it looks like the jacoco workaround was not correctly removed in #3293 but it kept working because the lobby subproject still had this workaround and therefore the required XML file was no longer generated, just the required html files.
I found a way to have it being generated and uploaded, which _seems_ to work, but I don't think this is the intended way.
Perhaps @ssoloff has an idea on how to proceed from here

Reply:
Sounds like we should restore both of the jacoco blocks. I notice this PR and recent ones do not have the automated codecov report, I suspect the now missing jacoco is why.

We're at a bit of a deadlock on code coverage: https://github.com/triplea-game/triplea/issues/3188. 
It might be reasonable for us to decide on a strict YAGNI  policy for the build configuration (which would mean I should update #3300 to not publish teh shadow jar until we strictly need it)

Reply:
I don't think restoring the jacoco blocks will solve the problem.
See https://github.com/triplea-game/triplea/commit/4fc0b5d04c465b450032303a1a3122b328a527f1 where @ssoloff added this block as a workaround.
I think the real root cause of this problem is the multi-project setup, we just didn't notice it earlier because of the workaround.

Reply:
Why would you say multi-project setup is root cause when the workaround was added well before the multi-project setup? I'd say the root cause was junit5 needed a jacoco workaround to begin with. The multi-project setup seems to have done nothing to change that picture, hence why it seems we *should* include the jacoco workaround blocks at all.

Reply:
I don't think we noticed it because code coverage is generally ignored, nothing requires us to look at it or have it.

Reply:
@DanVanAtta I just got it working 🎉